standard/lexical-structure.md

```diff
@@ -703,6 +752,25 @@ If the magnitude of the specified literal is too large to be represented in the
 The value of a real literal of type `float` or `double` is determined by using the IEC 60559 “round to nearest” mode with ties broken to “even” (a value with the least-significant-bit zero), and all digits considered significant.
 
 > *Note*: In a real literal, decimal digits are always required after the decimal point. For example, `1.3F` is a real literal but `1.F` is not. *end note*
+>
+> *Example*:
+>
+> ```csharp
+> 1.234_567              // double
+> .3e5f                  // float
+> 2_345E-2_0             // double
+> 15D                    // double
+> 19.73M                 // decimal
+> 1.F                    // invalid; ill-formed (parsed as "1." and "F")
```

Comment:
The comment is incorrect.  See https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQMwAJboMLIN7LpGYZQAs6AsgBQCUhxBSxL6AbgIYBO6H6AvOjgA6AGZ0A3AyIBfZHJSk4ANkwx0AUQAeAFwCmAOwDOASwD2x/NJKYV6EwZ3pxOgBYmj9x+gA2e0Tq06HiYAOxCEugKCkA=

Reply:
Change 'invalid; ill-formed (parsed as "1." and "F")' to something like 'not a decimal literal, as decimal points must be followed by at least one digit, but a member access'

Reply:
Done.